google/api_core/exceptions.py

```diff
@@ -22,12 +22,15 @@
 from __future__ import unicode_literals
 
 import http.client
+from google.rpc import error_details_pb2
 
 try:
     import grpc
+    from grpc_status import rpc_status
```

Comment:
Do we know for sure that `grpc_status` is compatible with the oldest version of `grpcio` we support?  If not, this import needs to be inside its own `try: .. except ImportError:` block, and we would also need to add it conditionally as another extra in `setup.py`.